processor/elasticapmprocessor/internal/ecs/ecs_translation.go

```diff
@@ -80,15 +91,25 @@ func isSupportedAttribute(attr string) bool {
 		string(semconv.CloudAccountIDKey),
 		string(semconv.CloudRegionKey),
 		string(semconv.CloudAvailabilityZoneKey),
-		string(semconv.CloudPlatformKey):
+		string(semconv.CloudPlatformKey),
+		"cloud.origin.account.id",
+		"cloud.origin.provider",
+		"cloud.origin.region",
+		"cloud.origin.service.name",
+		"cloud.account.name",
+		"cloud.instance.id",
+		"cloud.instance.name",
+		"cloud.machine.type",
+		"cloud.project.id",
+		"cloud.project.name":
 		return true
 
 	// container.*
 	case string(semconv.ContainerNameKey),
 		string(semconv.ContainerIDKey),
 		string(semconv.ContainerImageNameKey),
-		"container.image.tag",
-		"container.runtime":
+		string(semconv.ContainerImageTagsKey),
```

Comment:
this is semantically different as it translates to `container.image.tags` - oversight or why is `container.image.tag` not needed anymore?

Reply:
That was an oversight on my part. I see`container.image.tag` is used in the apm-data library and `semconv.ContainerImageTagsKey` is used in the `elasticapmintakereceiver`. I updated to support both for now

Reply:
thanks!